presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1786,4 +1787,17 @@ public FeaturesConfig setQueryOptimizationWithMaterializedViewEnabled(boolean va
         this.queryOptimizationWithMaterializedViewEnabled = value;
         return this;
     }
+
+    public boolean isAggIfToFilterRewriteEnabled()
+    {
+        return aggIfToFilterRewriteEnabled;
+    }
+
+    @Config("agg-if-to-filter-rewrite-enabled")
```

Comment:
we usually add a prefix "optimizer." to optimizer related config changes; check other examples in this class